pontoon/administration/files.py

```diff
@@ -64,27 +77,56 @@ def split_comments_mine(
     if (not end or (end > pointer)) and len(text) > pointer:
         cls.split_entities(text, object, code=code, pointer=pointer)
 
-PropertiesParser.split_comments = split_comments_mine
+PropertiesParser.split_comments = split_comments_pro
+
+
+@classmethod
+def split_comments_ini(cls, text, object, pointer=0, end=None):
+    pattern = cls.patterns['comment']
+    if end:
+        match = pattern.search(text, pointer, end)
+    else:
+        match = pattern.search(text, pointer)
+    while match:
+        st0 = match.start(0)
+        if st0 > pointer:
+            cls.split_entities(text, object, pointer=pointer, end=st0)
+        comment = Comment()
+        comment.add(
+            match.group(0)[1:].replace('\n;', '\n').replace('\n#', '\n'))
+        object.append(comment)
+        pointer = match.end(0)
+        if end:
+            match = pattern.search(text, pointer, end)
+        else:
+            match = pattern.search(text, pointer)
+    if (not end or (end > pointer)) and len(text) > pointer:
+        cls.split_entities(text, object, pointer=pointer, end=end)
+
+IniParser.split_comments = split_comments_ini
+
+IniParser.patterns['entity'] = re.compile(
```

Comment:
Same thing as above (bug 1017849).

I'm aware this is a hack and initially I thought it will be temporary. ;) I was hoping Silme will get fixed soon with my patch, but it didn't happen. I tried switching to [translate toolkit](http://toolkit.translatehouse.org/), but it also doesn't work entirely as expected, so I would need to patch it, too.